AzureCalling.xcodeproj/project.pbxproj

```diff
@@ -7,12 +7,8 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
-		1F25182425EF00C3004A9143 /* JoinCallType.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1F25182325EF00C3004A9143 /* JoinCallType.swift */; };
-		1F25182625EF4CEB004A9143 /* AttributedStringFactory.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1F25182525EF4CEB004A9143 /* AttributedStringFactory.swift */; };
 		1F5FAD8E25FAC62900E2365F /* CommunicationIdentifierExtension.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1F5FAD8D25FAC62900E2365F /* CommunicationIdentifierExtension.swift */; };
 		1F5FADB225FFC76A00E2365F /* ParticipantsEventsAdapter.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1F5FADB125FFC76A00E2365F /* ParticipantsEventsAdapter.swift */; };
-		1F7AFED325D5A71C00E0029C /* JoinCallConfig.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1F7AFED225D5A71C00E0029C /* JoinCallConfig.swift */; };
```

Comment:
We should continue to use JoinCallConfig.swift. It will be easier to merge later.